content/applications/inventory_and_mrp/manufacturing/basic_setup/mo_costs.rst

```diff
@@ -46,40 +46,57 @@ the component's product form, and entering a value. However, any future |POs| fo
-is entered in the :guilabel:`per employee` field, it costs $25.00 per hour for *each* employee
-working at the work center.
+:guilabel:`General Information` tab.
 
-Note that, if values are entered in both the :guilabel:`per workcenter` *and* :guilabel:`per
-employee` fields, the value in the :guilabel:`per workcenter` field takes precedence, and the value
-in the :guilabel:`per employee` field is ignored.
+To set the hourly cost of each employee that operates the work center, enter a value in the
+:guilabel:`per employee` field, located beside the :guilabel:`Cost per hour` section on the work
+center's :guilabel:`General Information` tab. For example, if `25.00` is entered in the
+:guilabel:`per employee` field, it costs $25.00 per hour for *each* employee working at the work
+center.
 
 .. important::
-   It is also possible to set a per hour cost for specific employees, by navigating to the
-   :menuselection:`Employees` app, selecting an employee, clicking the :guilabel:`HR Settings` tab
-   on their employee form, and entering a value in the :guilabel:`Hourly Cost` field.
+   The value entered in the :guilabel:`per employee` field is only used to calculate the |MO| cost,
+   which is the estimated cost of completing the |MO|.
+
+   The actual cost of completing the |MO| is represented by the real cost. Instead of using the
+   value entered in the :guilabel:`per employee` field, the real cost is calculated using the hourly
+   cost specific to each employee.
+
+   For example, if the :guilabel:`per employee` cost of a work center is '$50.00', and an employee
+   with an hourly cost of '$60.00' completes a work order there, the |MO| cost (estimated) is
+   calculated using the $50/hr cost, while the real cost is calculated using the $60/hr cost.
 
-   Just like the *per workcenter* field on a work center form, the :guilabel:`Hourly Cost` field on
-   an employee's form overrides the *per employee* field on a work center form.
+   See the :ref:`section below <manufacturing/mo-costs/employee-cost>` for information on how to set
+   the cost for specific employees.
 
-   However, the *per workcenter* field takes precedence over both the *per employee* field on the
-   workcenter form *and* the :guilabel:`Hourly Cost` field on the employee form.
+.. _manufacturing/mo-costs/employee-cost:
+
+Employee cost
+-------------
+
+To set the hourly cost for a specific employee, navigate to the :menuselection:`Employees` app, and
+select an employee. On the employee's form, select the :guilabel:`Settings` tab, and enter the
+employee's rate in the :guilabel:`Hourly Cost` field of the :guilabel:`Application Settings`
+section.
+
+.. important::
+   As detailed in the :ref:`section above <manufacturing/mo-costs/work-center-cost>`, the value
+   entered in the :guilabel:`Hourly Cost` field on the employee's form is used to calculate the real
+   cost of an MO. The estimated cost of an MO, referred to as the MO cost, uses the per employee
```

Comment:
```suggestion
   cost of an MO. The estimated cost of an MO, referred to as the MO cost, uses the :guilabel:`per employee`
```

Reply:
I don't think this needs to be italicized since it's not currently on the screen.